opencensus/plugins/grpc/internal/filter.h

```diff
@@ -104,6 +105,13 @@ trace::Span SpanFromCensusContext(const census_context *ctxt);
 // Returns a string representation of the StatusCode enum.
 absl::string_view StatusCodeToString(grpc_status_code code);
 
+inline const char *GetMethod(grpc_slice *path) {
```

Comment:
Please split this and related changes out into another PR.  This is the Sent.* Recv.* work, right?

Reply:
Done.